### PR TITLE
port to jbuilder

### PR DESCRIPTION
**Pull request by avsm**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,17 @@
-all:
-	ocaml pkg/pkg.ml build -n mirage-net
-	ocaml pkg/pkg.ml build -n mirage-net-lwt
+
+.PHONY: build clean test
+
+build:
+	jbuilder build @install
```
Comment (samoht) [inline Makefile]: can you add `--dev` here?

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+
+test:
+	jbuilder runtest
```
Comment (samoht) [inline Makefile]: can you add `--dev` here?

File: Makefile (hunk 1 above)
Comment (avsm) [reply Makefile]: done in 9a0292015eb111e569cfae22ce9d5759bcd59f35

File: Makefile (hunk 2 above)
Comment (avsm) [reply Makefile]: in  1707c1b2e26a8583a646c86e44929b43e60df850

File: lwt/.merlin [hunk 3]
```diff
@@ -1,6 +1,6 @@
 B ../_build/default/lwt
```
Comment (samoht) [inline lwt/.merlin]: can you remove the autogenerated merlin files too?

Comment (avsm) [reply lwt/.merlin]: Aren't we keeping those in the repo elsewhere?

Comment (avsm) [reply lwt/.merlin]: As in, I've checked them in on mine -- @djs55 do you gitignore merlin files?

Comment (samoht) [reply lwt/.merlin]: I am getting used to add them to .gitignore

the less auto-generated noise we have the better I think.

Comment (djs55) [reply lwt/.merlin]: Yes, I've been routinely adding `*.install` and `*.merlin` to `.gitignore`

Comment (avsm) [reply lwt/.merlin]: ok ...fixed in 8e34806fd6b596de1d8da3bca168e27b04dbf8dc

**Comment by djs55** [comment]
LGTM

**Comment by hannesm** [comment]
how comes after the discussion in #7 that there is now a `VERSION` file in the repository?

**Comment by djs55** [comment]
Oops -- I didn't spot that. I don't think the `VERSION` file is necessary -- I think `topkg tag` will read `CHANGES.md` and then the tag ends up in the installed `META` file. I'll make a PR to remove it.

**Comment by hannesm** [comment]
@djs55 thanks!  it might be worth to check all the other jbuilder-ported repositories to not accumulate this garbage all over?

**Comment by djs55** [comment]
@hannesm good idea, I'll put that on my TODO list

**Comment by djs55** [comment]
(I'm trying to maintain a per-week TODO list so I don't forget to do things)

**Pull request merged**